app/src/main/java/com/kickstarter/viewmodels/BackingFragmentViewModel.kt

```diff
@@ -181,6 +181,7 @@ interface BackingFragmentViewModel {
 
             backing
                     .map { it.backerName() }
+                    .filter { ObjectUtils.isNotNull(it) }
```

Comment:
Prevents calling the subscriber if backerName is null

Reply:
👍